test/unit/TestEval.hs

```diff
@@ -238,6 +239,23 @@ testEval g =
                  in T.concat ["(", t1, ",", t2, ") == split (chars ", t1, ") (", t1, " ++ ", t2, ")"]
                       `evaluatesToP` VBool True
             )
+        , testProperty
+            "char"
+            $ \i (NonEmpty s) ->
+              let i' = i `mod` length s
+               in T.concat ["char ", from @String (show i'), " ", tquote s]
+                    `evaluatesToP` VInt (fromIntegral (ord (s !! i')))
+        , testCase
+            "mkText 0"
+            ("mkText 0 (\\x. 97)" `evaluatesTo` VText "")
+        , testCase
+            "mkText 5"
+            ("mkText 5 (\\x. x+97)" `evaluatesTo` VText "abcde")
```

Comment:
I would prefer using `let ai = charAt 0 "a" in` than magic constants. Though hopefully, Unicode is the last encoding we will ever need. 😄

Reply:
Good idea. :heavy_check_mark: